### PR TITLE
bug 1533645 correcting signature transport example

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1533645
@thom-at-redhat PTAL

### CODE REVIEW

**Comment by wanghaoran1988** [comment]
LGFM

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL :white_check_mark:

File: security/deployment.adoc [hunk 1]
```diff
@@ -128,34 +128,39 @@ There are two types of signature transports.
...
+Signatures that use the `docker` transport type are served by local file or web
+server. These signatures are more flexible: you can serve images from any 
+container registry and use an independent server to deliver binary
+signatures. According to the
+link:https://github.com/containers/image/blob/master/docs/signature-protocols.md[Signature access protocols],
+you access the signatures for each image directly; the root of the server
+directory does not display its file structure.
+
+However, the `docker` transport type requires additional configuration. You must
+configure node with the URI of the signature server by placing
```
Comment (mburke5678) [inline security/deployment.adoc]: s/configure node/configure the node (or configure nodes)?

File: security/deployment.adoc [hunk 2]
```diff
@@ -128,34 +128,39 @@ There are two types of signature transports.
...
+arbitrarily-named YAML files into a directory on the host system,
```
Comment (mburke5678) [inline security/deployment.adoc]: This is a legacy issue, but do you know if the "arbitrarily-named" YAML file same as the "configuration files" that contain the URI/sigstore?

File: security/deployment.adoc [hunk 3]
```diff
@@ -128,34 +128,39 @@ There are two types of signature transports.
...
-This example file may be placed on each node as
-*_/etc/containers/registries.d/redhat.com.yaml_*. It defines a signature server
-(the `sigstore` parameter) to serve signatures for the `docker` transport type
-for the Red Hat Registry (`access.registry.redhat.com`). Placing these files
-onto a cluster of nodes may be automated using Ansible, for example. No service
+In this example, the Red Hat Registry, `access.redhat.com`, is the signature
+server that provides signatures for the `docker` transport type. Its URI is
+defined in the `sigstore` parameter. You might name this file 
+*_/etc/containers/registries.d/redhat.com.yaml_*, and use Ansible to
+automatically place the file on each node in your cluster. No service
```
Comment (mburke5678) [inline security/deployment.adoc]: Do we need the comma here: *_/etc/containers/registries.d/redhat.com.yaml_*, and

**Comment by mburke5678** [comment]
@kalexand-rh A couple of nits and one question.

File: security/deployment.adoc (hunk 1 above)
Comment (kalexand-rh) [reply security/deployment.adoc]: I think "configure the nodes" works best.

File: security/deployment.adoc (hunk 2 above)
Comment (kalexand-rh) [reply security/deployment.adoc]: That's the way I read it. I can change "configuration files" in the next sentence to "YAML configuration files," if you think that would help.

File: security/deployment.adoc (hunk 3 above)
Comment (kalexand-rh) [reply security/deployment.adoc]: Nope. Thanks!

**Comment by kalexand-rh** [comment]
@mburke5678, I fixed the nits and hopefully addressed your question. Is this acceptable?

**Pull request merged**